### ISSUE TITLE
Varshini Prakash - Persuasion - Policy-specific (21)

### ISSUE THREAD

**Issue by ranimurali412** [ranimurali412/code4policy-team-a1#2]
**As the** leader of an advocacy organisation influencing policymakers/government 

**I want to** ensure we have the evidence base on what climate policies are most inequitable

**So that I** can make a persuasive argument to influence new equitable policies

-----

Definitions of done:

- [x] user is able to clearly see policies included in Biden's Climate Plan and identify the 10 most inequitable climate policies (in terms of workforce)

---

Tasks:

- [x] add section on what's in Biden's climate plan
- [x] visualization on most inequitable climate policies

**Comment by rosekhattar** [comment]
Perhaps we should change "right tools" to an "evidence base"?

**Comment by amandasaid** [comment]
I agree - I think we should be more specific in the "I want" section

**Comment by Emily-coding** [comment]
I like that too. Perhaps we could also change the first persuading to influencing?

**Comment by ranimurali412** [comment]
Agreed! will change

**Comment by amandasaid** [comment]
This one can be more policy-specific (we will need to workshop)
